.github/workflows/actions.yml

```diff
@@ -0,0 +1,46 @@
+name: repo-checks
+
+on:
+  pull_request:
+    branches:
+    - master
+    - deploy
+
+jobs:
+  main:
+    name: rvm-matrix
+    runs-on: ubuntu-latest
+    env:
+      RAILS_ENV: test
+    continue-on-error: ${{ matrix.allow_failures }}
+    strategy:
+      fail-fast: true
+      matrix:
+        allow_failures: [false]
+        rvm:
+          - 2.4
+          - 2.5
+          - 2.6
+        include:
+          - rvm: head
+            allow_failures: true
+    steps:
+    - uses: zendesk/checkout@v2
+    - uses: zendesk/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.rvm }}
+    - name: ruby ${{ matrix.rvm }}
+      run: |
+        gem install rake -v 13.0.0
+        gem install activesupport -v 4.2.11.1 
+        gem install builder -v 3.2.3
+        gem install crass -v 1.0.5 
+        gem install loofah -v 2.3.1
+        gem install rails-html-sanitizer -v 1.3.0
+        gem install actionview -v 4.2.11.1
+        gem install rack -v 1.6.11
+        gem install actionpack -v 4.2.11.1 
+        gem install bump -v 0.8.0
+        gem install wwtd -v 1.4.0
```

Comment:
no need for that, `bundle install` should take care of that

Reply:
it was not finding these gems with bundle install. let's see if bundle update would do it.